homeassistant/components/ps4/media_player.py

```diff
@@ -58,11 +58,6 @@
 async def async_setup_entry(hass, config_entry, async_add_entities):
     """Set up PS4 from a config entry."""
     config = config_entry
-    await async_setup_platform(hass, config, async_add_entities, discovery_info=None)
```

Comment:
Maybe add a notification or raise exception if async_setup_platform is called? Still would get a error.

`AttributeError: module 'homeassistant.components.ps4.media_player' has no attribute 'setup_platform'`

Reply:
We usually implement it but just `pass`.